domainbed/algorithms.py

```diff
@@ -1007,22 +1007,23 @@ def update(self, minibatches, unlabeled=False):
             env_loss = F.cross_entropy(logits, y)
             total_loss += env_loss
 
-            grads.append( autograd.grad(env_loss, self.network.parameters(), retain_graph=True) )
+            grads.append( autograd.grad(env_loss, self.network.parameters(), retain_graph=True, create_graph=True) )
```

Comment:
Remove trailing whitespaces and respect max 80 characters per line. Remove unnecessary `retain_graph=True`.